codegen/method.go

```diff
@@ -63,6 +63,8 @@ type MethodSpec struct {
 	RequestStruct []StructSpec
 	// Thrift service name the method belongs to.
 	ThriftService string
+	// The thriftrw-generated go package name
+	GenCodePkgName string
 	// Whether the method needs annotation or not.
 	WantAnnot bool
```

Comment:
The Method may or may not **need** the thrift file to have annotations to be created, hence the verb